Gems/Multiplayer/Code/Source/MultiplayerSystemComponent.cpp

```diff
@@ -489,11 +489,23 @@ namespace Multiplayer
     {
         m_didHandshake = true;
 
-        AZ::CVarFixedString commandString = "sv_map " + packet.GetMap();
-        AZ::Interface<AZ::IConsole>::Get()->PerformCommand(commandString.c_str());
+        // If this is an Editor then we're now accepting the connection to the EditorServer.
+        // In normal game clients SendReadyForEntityUpdates will be enabled once the appropriate level's root spawnable is loaded,
+        // but since we're in Editor, we're already in the level.
+        AZ::ApplicationTypeQuery applicationType;
+        AZ::ComponentApplicationBus::Broadcast(&AZ::ComponentApplicationRequests::QueryApplicationType, applicationType);
+        if (applicationType.IsEditor())
+        {
+            SendReadyForEntityUpdates(true);
+        }
+        else
+        {
+            AZ::CVarFixedString commandString = "sv_map " + packet.GetMap();
+            AZ::Interface<AZ::IConsole>::Get()->PerformCommand(commandString.c_str());
 
-        AZ::CVarFixedString loadLevelString = "LoadLevel " + packet.GetMap();
-        AZ::Interface<AZ::IConsole>::Get()->PerformCommand(loadLevelString.c_str());
+            AZ::CVarFixedString loadLevelString = "LoadLevel " + packet.GetMap();
+            AZ::Interface<AZ::IConsole>::Get()->PerformCommand(loadLevelString.c_str());
+        }
```

Comment:
I'm not sure an if else here is correct. The Editor should still need the LoadLevel logic since it will try to "LoadLevel" a root.spawnable that is generated by the Ctrl+G process. 

I'd also see if you can find a more appropriate home for calling SendReadyForEntityUpdates(true) in MultiplayerEditorSystemComponent instead of in MultiplayerSystemComponent.

Reply:
_"I'm not sure an if else here is correct. The Editor should still need the LoadLevel logic since it will try to "LoadLevel" a root.spawnable that is generated by the Ctrl+G process."_
In what use case is that needed?  Hmm, are you saying if I set editorsv_launch = false and instead have my own server running, then when I press CTRL+G in the editor I shouldn't run the level i'm currently in my editor with and instead use the map the server is in?  I haven't tried that use case, but I'd certainly expect the server switch to using the Editor's current level.

"I'd also see if you can find a more appropriate home for calling SendReadyForEntityUpdates(true) in MultiplayerEditorSystemComponent instead of in MultiplayerSystemComponent."
Since MultiplayerEditorSystemComponent  doesn't actually listen for packets, this would really change the class but could be done.  It'll require updating EditorSystemComponent to become an IConnectionListener and creating/registering a new network interface.  MultiplayerEditorConnection is already a IConnectionListener, but this is shutdown once the editorserver is ready.

Reply:
I wouldn't make it an IConnectionListener. I'd just grab the component and call a function on it or create an AZ::Event. If there's a non-network event to listen that's an option too. MultiplayerEditorSystemComponent is closely tied to this component so its existence is a good thing to check against is the point.

What's happening here with Ctrl+G is the following
- Ctrl+G is triggered, Editor creates an in-memory spawnable called root.spawnable representing the current Editor world and registers that asset
- MultiplayerEditorConnection in tandem with MultiplayerEditorSystemComponent serializes and sends that spawnable to a server application
- The server registers that asset and then loads it as a level
- MultiplayerEditorConnection hands the Editor off as a Client to the Server and disconnects itself, we're now in the normal Multiplayer flow
- On Accept, Multiplayer loads the level indicated by the server. In this case, root.spawnable so the load level should here should not be cased at all

Note that because of this, Editor as a Client via Ctrl+G is limited to one client connected to the server as other clients would have no context on the in memory root.spawnable.